app/rtc_s1.py

```diff
@@ -369,8 +363,12 @@ def run(cfg: RunConfig):
         product_id = 'rtc_product'
     product_prefix = f'{product_id}_v{product_version}'
 
-    scratch_path = os.path.join(
-        cfg.groups.product_group.scratch_path, f'temp_{time_stamp}')
+    if tempdir:
+        scratch_path = tempdir
+    else:
+        scratch_path = os.path.join(
+            cfg.groups.product_group.scratch_path, f'temp_{time_stamp}')
```

Comment:
Why can't you set `scratch_path` from the child's process runconfig? Please review pseudo-code that we discussed in slack.